### DISCUSSION TITLE
Simpsons theme: White font difficult to read over white clouds

### DISCUSSION THREAD

**Discussion by Tatwi** [OnionUI/Onion#1610] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

v4.3.0

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

I expected to be able to read all the menu/game listings clearly, but instead only the selected listing has a drop shadow on the text. As such, the white text gets lost in the white clouds in the background image.

### Steps to Reproduce

Steps:
1. Install ONION Simpsons theme.
2. Notice that every menu item and game name on the lower 2/3 of the screen is obfuscated by the clouds in the background, except for the currently selected line.

### Provide any additional context or information that may help us investigate your issue

A simple solution would be to add the drop shadow to all of the text, that way the white text will stand out against the white clouds without diminishing the overall aesthetic.

My wife and I *love* this theme, but this issue makes it unusable for us, because it makes reading the game lists and other menus very annoying. We like this theme so much, I am taking the time to provide this feedback! :)

**Comment by Aemiii91** [comment]
Can you please attach a screenshot?

**Comment by Tatwi** [comment]
![MainUI_000](https://github.com/OnionUI/Onion/assets/9004870/fc3c939f-1a48-4555-b90c-7baf043d6714)
![MainUI_001](https://github.com/OnionUI/Onion/assets/9004870/8bb3dc8d-ae8d-474d-84d0-c3a63cb4a20d)
![MainUI_002](https://github.com/OnionUI/Onion/assets/9004870/c4a35ccc-0eff-4b83-aedd-ae85e49cd87c)
![MainUI_003](https://github.com/OnionUI/Onion/assets/9004870/ee64e8f9-1a38-4099-88c0-9406a3582d8e)

**Reply by Aemiii91** [reply]
We can't add drop shadow (the frontend is closed source). We are actually removing the existing drop shadow in v4.3.1

I would say the solution would be to change the text color of the theme, or removing the clouds.

**Reply by Tatwi** [reply]
The clouds are charming, nostalgic, and character agnostic, so it would be sad to see them go. Simpsons-skin-tone yellow might look ok for a font color and it shouldn't get lost in either the white or the blue, but it would change the feel of theme. Too bad about the drop shadows, they are a very practical solution to the visibility of fonts in general.
